src/components/toggle-group/_editor.scss

```diff
@@ -1,12 +1,20 @@
+.components-toggle-group {
+	overflow: hidden;
+}
+
+.components-toggle-group__toggle-list {
```

Comment:
On animated elements from toggle-group we should add `will-change: opacity, transform` to improve performance.